realm/realm-jni/project.mk

```diff
@@ -52,7 +52,7 @@ ifeq ($(REALM_ANDROID),)
     endif
   endif
 else
-  PROJECT_CFLAGS += -fvisibility=hidden -DANDROID
+  PROJECT_CFLAGS += -fvisibility=hidden -DANDROID -ffunction-sections -fdata-sections -flto
```

Comment:
I added them for consistency. I hope we can move to a "pure" Gradle/Android Studio setup soon.